interns-react-weekly-assignments/src/components/Home/Ritika/Week3/FriendStatus/FormInputField/FriendStatus.styled.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+
+export const FriendStatusWrapper = styled.div`
+  background-color: #f0f8ff;
+  height: 800px;
+  width: 70%;
+  border: 2px solid #000000;
+  margin: 12px auto;
+
+  .headings {
+    padding: 10px;
+    margin: 4px 0;
+    font-size: 18px;
+  }
+
+  .inputFields {
+    padding: 10px;
+    margin: 5px 0;
+    width: 100%;
+    border: 1px solid #cccccc;
+    border-radius: 5px;
+    font-size: 16px;
+  }
+
+  .submitButton {
+    padding: 10px;
+    margin: 8px 0;
+    width: 100%;
+    background-color: #007bff;
+    color: #ffffff;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 16px;
+  }
+
+  .mainContainer {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    margin: auto;
+    margin-top: 30px;
+  }
+  @media screen and (max-width: 576px) {
+    .AutoFocusWrapper {
```

Comment:
pascal....

Reply:
It is wrapper we need to name it in upper Pascal only.